### ISSUE TITLE
Vertical plants do not drop the blocks below the one that was broken

### ISSUE THREAD

**Issue by Direbat** [TheCyberBrick/The-Betweenlands#1278]
#### Betweenlands version:

3.7.3

#### Forge version:

14.23.5.2854

#### Singleplayer or Multiplayer:

Happens in single player, have not tested multiplayer

#### Installed mods:

Into the Betweenlands modpack version 1.3.5
[Into The Betweenlands-1.3.5.zip](https://github.com/Angry-Pixel/The-Betweenlands/files/7212281/Into.The.Betweenlands-1.3.5.zip)

#### How to reproduce:

1.Get syrmorite shears
2. break block of hanger with more blocks beneath it
3. break block of poison ivy with more blocks beneath it
4. break block of cave moss with more blocks beneath it

#### Expected result:

All the vertical plants would drop the blocks below the one broken similar to how swamp reeds work

#### Actual result:

Only the block that was broken by the shears was dropped, the rest just disappeared

#### Pictures:

![2021-09-22_21 49 38](https://user-images.githubusercontent.com/74643395/134383272-430f4572-0145-4094-b5b9-f3bf64e9ddbb.png)
![2021-09-22_21 49 43](https://user-images.githubusercontent.com/74643395/134383278-3219d470-1be1-41fe-9ead-58d2df27032c.png)

**Comment by PetasiGit** [comment]
They work like regular vanilla vines, I personally don't see an issue with it. This is more of a suggestion for their behavior.

**Comment by Direbat** [comment]
Yes but you normally don't have to collect a large amount of vanilla vines for crafting, but you sometimes do for these plants(especially for caving rope) so I still think it is nice for at least hangers to drop the block below just to make collecting large amounts easier.

**Comment by PetasiGit** [comment]
You can craft caving rope from swamp reeds too. But I understand what you're saying.

**Comment by Direbat** [comment]
Wait you can? JEI scammed me.
